lua/nvim-treesitter/parsers.lua

```diff
@@ -1480,6 +1480,18 @@ list.smali = {
   maintainers = { "@amaanq" },
 }
 
+list.snakemake = {
+  install_info = {
+    url = "https://github.com/osthomas/tree-sitter-snakemake",
+    branch = "main",
```

Comment:
```suggestion
```

No need to specify the default branch. (It's enough to have a lockfile entry then.)